content/en/docs/tasks/security/authz-http/index.md

```diff
@@ -32,178 +30,259 @@ the product page, you can see the following sections:
+spec:
+  rules:
+  - from:
+    - source:
+        namespaces: ["default", "istio-system"]
+    to:
+    - operation:
+        methods: ["GET"]
+EOF
+{{< /text >}}
 
-{{< tip >}}
-There may be some delays due to caching and other propagation overhead.
-{{< /tip >}}
+You can expect to see output similar to the following:
 
-## Cleanup
+{{< text plain >}}
+authorizationpolicy.security.istio.io/bookinfo-viewer created
+{{< /text >}}
 
-*   Remove Istio authorization policy configuration:
+Point your browser at Bookinfo's `productpage` (`http://$GATEWAY_URL/productpage`).
+You should see the "Bookinfo Sample" page, with the "Book Details" section in the lower left part
+and the "Book Reviews" section in the lower right part.
 
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="remove_istio_authorization_policy.sh" >}}
+**Congratulations!** You successfully deployed applied a namespace-level authorization policy
+to enforce access control for the requests.
 
-    Alternatively, you can delete all `ServiceRole` and `ServiceRoleBinding` resources by running the following commands:
+### Clean up namespace-level access control
 
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="remove_istio_authorization_policy_alternative.sh" >}}
+Remove the authorization policy with the following command before continuing:
 
-*   Disable Istio authorization:
+{{< text bash >}}
+$ kubectl delete authorizationpolicy.security.istio.io/bookinfo-viewer
+{{< /text >}}
 
-    {{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="disabling_istio_authorization.sh" >}}
+## Enforce workload-level access control
+
+This task shows you how to set up workload-level access control using Istio authorization.
+You will start with a simple `deny-all` policy that rejects all requests to the workload, and then
+grant the permission gradually and incrementally to allow more access to the workload.
+
+1. Apply a default `deny-all` policy for workloads in the default namespace
+
+    The following command creates a `deny-all` policy with no workload selector, the policy will
+    select all workloads in the default namespace. The policy will deny all access and we will
+    add more policies to grant the access incrementally in following steps.
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: deny-all
+    spec:
+      {}
+    EOF
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`).
+    Now you should see `"RBAC: access denied"`. The error shows that the configured `deny-all` policy
+    is working as intended, and Istio doesn't have any rules that allow any access
+    to workloads in the default namespace.
+
+1. Allow access to the `productpage` workload
+
+    The following command creates a `productpage-viewer` policy which allows
```

Comment:
Please make sure the explanation comes before the call to action.
Readers need the context before you ask them to perform the action, not after.